pkg/pgmodel/migrations/sql/1_base_schema.up.sql

```diff
@@ -693,8 +699,11 @@ GRANT EXECUTE ON FUNCTION SCHEMA_CATALOG.create_series(int, name, SCHEMA_PROM.la
 
 CREATE OR REPLACE  FUNCTION SCHEMA_PROM.series_id(label jsonb)
```

Comment:
Given all of these locking issues, I'd want a get only version of this function I think that didn't try and create any of this infrastructure and wouldn't accidentally end up locking (or not working on a replica?)

Reply:
Also, I thought we were saying that all of the volatile functioins weren't going to be in the public api essentially? Because they shouldn't be run by users?